[2022-04-26 01:19:58,043] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: starwarapi.transform_load scheduled__2022-04-26T00:49:49.654927+00:00 [queued]>
[2022-04-26 01:19:58,986] {taskinstance.py:1043} INFO - Dependencies all met for <TaskInstance: starwarapi.transform_load scheduled__2022-04-26T00:49:49.654927+00:00 [queued]>
[2022-04-26 01:19:59,002] {taskinstance.py:1249} INFO - 
--------------------------------------------------------------------------------
[2022-04-26 01:19:59,003] {taskinstance.py:1250} INFO - Starting attempt 1 of 2
[2022-04-26 01:19:59,012] {taskinstance.py:1251} INFO - 
--------------------------------------------------------------------------------
[2022-04-26 01:19:59,454] {taskinstance.py:1270} INFO - Executing <Task(PythonOperator): transform_load> on 2022-04-26 00:49:49.654927+00:00
[2022-04-26 01:19:59,651] {standard_task_runner.py:52} INFO - Started process 174 to run task
[2022-04-26 01:19:59,698] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'starwarapi', 'transform_load', 'scheduled__2022-04-26T00:49:49.654927+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/starwar_dag.py', '--cfg-path', '/tmp/tmp1p9ki2cb', '--error-file', '/tmp/tmp24ejgp27']
[2022-04-26 01:19:59,746] {standard_task_runner.py:80} INFO - Job 33: Subtask transform_load
[2022-04-26 01:20:01,342] {logging_mixin.py:109} INFO - Running <TaskInstance: starwarapi.transform_load scheduled__2022-04-26T00:49:49.654927+00:00 [running]> on host 66327e1a8873
[2022-04-26 01:20:02,092] {taskinstance.py:1448} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=starwarapi
AIRFLOW_CTX_TASK_ID=transform_load
AIRFLOW_CTX_EXECUTION_DATE=2022-04-26T00:49:49.654927+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-26T00:49:49.654927+00:00
[2022-04-26 01:20:50,328] {python.py:175} INFO - Done. Returned value was: None
[2022-04-26 01:20:52,165] {taskinstance.py:1288} INFO - Marking task as SUCCESS. dag_id=starwarapi, task_id=transform_load, execution_date=20220426T004949, start_date=20220426T011958, end_date=20220426T012052
[2022-04-26 01:20:54,813] {local_task_job.py:212} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-04-26 01:20:55,293] {process_utils.py:124} INFO - Sending Signals.SIGTERM to group 174. PIDs of all processes in the group: [174]
[2022-04-26 01:20:55,334] {process_utils.py:75} INFO - Sending the signal Signals.SIGTERM to group 174
[2022-04-26 01:20:55,866] {process_utils.py:70} INFO - Process psutil.Process(pid=174, status='terminated', exitcode=0, started='01:19:59') (174) terminated with exit code 0
